pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "compass-sdk"
-version = "0.1.1"
+version = "0.2.0"
```

Comment:
@benrules3 how should we version this? Or do we work with animal name and add a release process?

Reply:
I believe installs are all done against the git url, so probably good as is for now